test/Graphics/ConvexShape.test.cpp

```diff
@@ -1,3 +1,4 @@
+#include <SFML/Graphics/CircleShape.hpp>
```

Comment:
`ConvexShape.hpp` must be the first included header. Look [here](https://github.com/SFML/SFML/blob/97c00d42adbc2934b09d6b6120f4188a80d4803c/test/Graphics/Texture.test.cpp#L4) to see how we're handling header order in tests.